lib/resolvers.js

```diff
@@ -0,0 +1,34 @@
+import dateFns from 'date-fns';
+
+export const mapDateTime = fieldName => doc => {
+  const timestamp = doc[fieldName];
+  const date = new Date(timestamp);
+
+  return {
+    iso: timestamp,
+    date: timestamp ? dateFns.format(date, 'DD') : null,
+    month: timestamp ? dateFns.format(date, 'MM') : null,
+    monthSym: timestamp ? dateFns.format(date, 'MMM') : null,
+    year: timestamp ? dateFns.format(date, 'YYYY') : null,
```

Comment:
Needed to check whether the timestamp value from Prismic was non-null here otherwise we'd end up with incorrect values for `date`, `month`, `monthSym` and `year` while iso was `null`.